src/common/utils.ts

```diff
@@ -0,0 +1,15 @@
+export const paramsQueryBuilder = (params: Record<string, unknown>): string => {
+  const queryStringArray: string[] = [];
```

Comment:
add // TODO: need to support nested paremeters, currently only 1 level is supported

Reply:
added